Resources/public/js/views/fields/ez-binarybase-editview.js

```diff
@@ -114,9 +114,14 @@ YUI.add('ez-binarybase-editview', function (Y) {
          */
         _getFieldValue: function () {
             var file = this.get('file'),
+                actualLanguageCode = this.get('languageCode'),
+                currentVersion = this.get('content').get('currentVersion'),
+                translationExist =  Y.Array.some(currentVersion.getTranslationsList(), function (languageCode) {
```

Comment:
I would put this logic in a dedicated method. Also, what you are checking here is not exactly that the translation exist but rather that the user is creating a new translation, so this method would be called `_isCreatingTranslation`.
Nitpick: you can also avoid using Y.Array.some by just using indexOf on the array.